### PR TITLE
anyone can rotate items in itemframe #to fix

### PR DESCRIPTION
**Pull request by SuperKali**

### CODE REVIEW

File: src/pocketmine/block/ItemFrame.php [hunk 1]
```diff
@@ -76,6 +76,7 @@ public function onActivate(Item $item, Player $player = null){
 		}
 
 		if($tile->hasItem()){
+			return;//anyone can rotate items in itemframe #to fix
 			$tile->setItemRotation(($tile->getItemRotation() + 1) % 8);
```
Comment (Leo3418) [inline src/pocketmine/block/ItemFrame.php]: If you `return` in the previous line, this line will become redundant and useless, won't it?

Comment (SuperKali) [inline src/pocketmine/block/ItemFrame.php]: But we are not on vanilla? If they use this on a server and useful

Comment (Leo3418) [inline src/pocketmine/block/ItemFrame.php]: Then they may use plugins.

**Comment by TDMidkwhatisxd** [comment]
i noticed that frames in my plots are not safe. Anyone can take item from it but cant put back, only if u have acces you can put a item in frame.

**Comment by DerddyDert** [comment]
@CookieCode solved all the frame issue but he disappeared...

**Pull request closed without merging**